hey.go

```diff
@@ -189,6 +193,15 @@ func main() {
 		req.Host = *hostHeader
 	}
 
+	// if request body is file
+	if *bodyPath != "" {
+		all, err := ioutil.ReadFile(*bodyPath)
+		if err != nil {
+			 log.Fatal(err)
+		}
+		*body = string(all)
```

Comment:
I wonder if we should modify RequestBody to be a byte slice rather than casting all to string.